docs/index.js

```diff
@@ -18,10 +21,11 @@ let Wrapper = React.createClass({
     })
   },
 
-  async crop () {
-    let image = await this.refs.crop.cropImage()
-    this.setState({
-      previewUrl: window.URL.createObjectURL(image)
+  crop () {
```

Comment:
This is only a docs file so it will have no effect on the component. `this.refs.crop` just returns a promise and so it's up to the consumer to decide how that promise is handled. We just happen to use async/await in this example but it can be consumed with just a `.then` as well. This component is also run through babel prior to being published to npm and thus any new feature being used here that you don't use in your project will have no effect.